### PR TITLE
Home Page implementation

### PR DESCRIPTION
**Pull request by willy1989cv**
Issue: https://github.com/transport-data/tdc-data-portal/issues/22

- Added hero asset configuration through a json file
- Added dynamic testimonials feature
- Fetching FAQ's from `/md/faq` folder
- Documentations

### CODE REVIEW

File: frontend/src/pages/data-provider.tsx [hunk 1]
```diff
@@ -21,10 +30,43 @@ export default function Home(): JSX.Element {
...
       </Layout>
     </>
   );
 }
+
+export const getServerSideProps: GetServerSideProps<any> = async () => {
+  const mddb = await clientPromise;
+
+  //fetch faqs
+  const faqsFiles = await mddb.getFiles({
```
Comment (demenech) [inline frontend/src/pages/data-provider.tsx:45]: Deployment is broken https://tdc-data-portal-git-feature-homepage-functional-impl-datopian1.vercel.app/data-provider due to the `getServerSideProps` with `fs` and `mddb` calls in it

File: frontend/src/pages/events.tsx [hunk 2]
```diff
@@ -30,6 +30,9 @@ export default function EventsPage({
   const [highlighted, ...rest] = upcoming;
   const otherEvents = rest?.splice(0, 2);
   const otherEventsSize = otherEvents.length;
+
+  console.log(upcoming);
```
Comment (demenech) [inline frontend/src/pages/events.tsx:34]: Remove console.log

**Review by demenech — CHANGES_REQUESTED** [review]

File: frontend/src/pages/data-provider.tsx (hunk 1 above)
Comment (willy1989cv) [reply frontend/src/pages/data-provider.tsx:15]: fixed

File: frontend/src/pages/events.tsx (hunk 2 above)
Comment (willy1989cv) [reply frontend/src/pages/events.tsx]: fixed. also improved cards display

**Review by demenech — APPROVED** [review]
LGTM

**Pull request merged**